elasticsearch/src/main/java/org/hibernate/search/elasticsearch/util/impl/GsonHttpEntity.java

```diff
@@ -169,14 +169,27 @@ public Header getContentEncoding() {
 	public InputStream getContent() throws IOException {
 		//This could be implemented but would be sub-optimal compared to using produceContent().
 		//We therefore prefer throwing the exception so that we can easily spot unintended usage via tests.
-		throw new UnsupportedOperationException( "Not implemented! Expected to produce content only over produceContent()" );
+		throw new UnsupportedOperationException( "Not implemented! Expected to produce content only over produceContent(),"
+				+ " or writeTo(OutputStream) if blocking calls are acceptable for your use case." );
 	}
 
 	@Override
-	public void writeTo(OutputStream outstream) throws IOException {
-		//This could be implemented but would be sub-optimal compared to using produceContent().
-		//We therefore prefer throwing the exception so that we can easily spot unintended usage via tests.
-		throw new UnsupportedOperationException( "Not implemented! Expected to produce content only over produceContent()" );
+	public void writeTo(OutputStream out) throws IOException {
+		/*
```

Comment:
Hum, yes, there's a risk of the entity being misused. Not sure it's a big one, but it exist nonetheless.
However, implementing this method was the point of this whole PR. If we decide we want to throw an exception here, it means this PR is rejected.
Since you marked the PR with the "Changes required" label, I guess your message was only meant as an observation?